attendance/database.py

```diff
@@ -0,0 +1,19 @@
+import psycopg2
+
+con = psycopg2.connect(dbname = 'vit', user = 'surya2', host = 'localhost', port = 26257)
+con.set_session(autocommit = True)
+
+cur = con.cursor()
+
+def getRow(query):
+    global cur
+    global con
+
+    cur.execute(query)
+
```

Comment:
remove redundant spaces, add spaces to separate some logical steps inside a function